README.md

```diff
@@ -1,45 +1,75 @@
+This ensures data transmissions with constant latency, regardless of the size of the payload. For more information have a look at the 
+[1000 words iceoryx introduction](https://www.eclipse.org/community/eclipse_newsletter/2019/december/4.php)
 
-Originating from the automotive domain, it is crucial to transfer a huge amount of data between multiple processes to
-realize driver assistance systems or automated driving applications. Moreover, the same efficient communication
-mechanism can be applied to a broader range of use cases, e.g. in the field of robotics or game development.
+<p align="center">
+<img src="https://user-images.githubusercontent.com/55156294/91751530-35af7f80-ebc5-11ea-9aed-eed590b229df.png" width="80%">
+</p>
 
-It's all about the API!
+You're right, middleware is a cluttered term and can somehow be all or nothing, so let's talk about the [goals and non-goals](doc/goals-non-goals.md) of iceoryx.
+
+It's all about the API?!
 
 Don't get too frighten of the API when strolling through the codebase. Think of iceoryx's API as a "plumbing" one
 ("plumbing" as defined in Git, which means low-level). We're not using the "plumbing" API ourselves, but instead a typed API.
-An example for a "porcelain" API would be [ROS2](https://www.ros.org/). Others are listed in the next section.
+The normal use case is that iceoryx is integrated as high-performance IPC transport layer in a bigger framework with additional API layers. 
+An example for such a "porcelain" API would be [ROS2](https://www.ros.org/). Others are listed in the next section.
 
-### Where is Eclipse iceoryx used?
+### Where is Eclipse iceoryx already used?
 
 |Framework | Description |
 |---|---|
-| [ROS2](https://github.com/ros2/rmw_iceoryx) | Eclipse iceoryx can be used inside the [robot operating system](https://www.ros.org/) with rmw_iceoryx |
-| [eCAL](https://github.com/continental/ecal) | Open-source middleware from [Continental AG](https://www.continental.com/) supporting pub/sub and various message protocols |
+| [ROS2](https://github.com/ros2/rmw_iceoryx) | Eclipse iceoryx can be used inside the [robot operating system](https://www.ros.org/) with [rmw_iceoryx](https://github.com/ros2/rmw_iceoryx.git) |
+| [eCAL](https://github.com/continental/ecal) | Open-source framework from [Continental AG](https://www.continental.com/) supporting pub/sub and various message protocols |
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [Adaptive AUTOSAR](https://www.autosar.org/standards/adaptive-platform/) platform software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
 | [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/) |
 
+## Installation
+
+You can find the installation guidelines [here](doc/installation-guide.md).
```

Comment:
If you like "Build & Install", then it should also be the "build and installation guide"